ManatiCR/FDI
# FDI-28: Create directorio content type.

[FDI-28](https://manati.atlassian.net/browse/FDI-28)
-
**Steps to test**
- Run `ahoy drush cim -y`
- Log in as an admministrador de contenido user
- Create one node directorio, check you can edit it and delete it as well
